floris/simulation/farm.py

```diff
@@ -81,8 +81,11 @@ class Farm(BaseClass):
 
     turbine_definitions: list = field(init=False, validator=iter_validator(list, dict))
 
-    turbine_fCts: Dict[str, interp1d] | List[interp1d] = field(init=False, factory=list)
-    turbine_fCts_sorted: NDArrayFloat = field(init=False, factory=list)
+    turbine_thrust_coefficient_functions: Dict[str, Callable] | List[Callable] = \
+        field(init=False, factory=list)
+
+    turbine_axial_induction_functions: Dict[str, Callable] | List[Callable] = \
```

Comment:
These were previously either a dict or a list depending whether it was a multidimensional turbine. Now I believe they're only dict. Is that true?